docs/README.md

```diff
@@ -47,10 +47,10 @@ Empathy in our team is highly valued.
 We always put ourselves into our reviewers position to improve their life when reviewing our code.
 
 - For each iteration, please create a branch with the following
-  pattern: `<firstname>_<lastname>_session_<session-nr>_iteration_<iteration-nr>_<pairing-partner-name>`
+  pattern: `<firstname>_<lastname>_iteration_<iteration-nr>_<pairing-partner-name>`
```

Comment:
`<pairing-partner-lastname>` instead of `<pairing-partner-name>` ?

Reply:
Why do you want to drop the session? For me it was much easier to check out the branches in the correct order when there was a session identifier in the branch name.

Reply:
I guess because of some script to create the PRs but why not adapt the script?
The idea was to make it easier to find the actual session along the iteration as I needed this quite often in the past and it was usually hard to find.

Reply:
> Introducing session is unnecessary as you see the progression by looking at the branches themself (e.g. they naturally stack).

That's not true, e.g. with this example you have no idea which branch comes first:

`iteration_1_berni`
`iteration_1_marton`
`iteration_2_rohini`

In which branch was the first iteration finished, in my or yours?

Reply:
Why do we even need so many branches?
Can't we just get one branch per hiring and review it per commit?
Usually i just mark all commits in intellij and review them as one iteration.
Only thing needed it that they prefix their commits with the iteration number.
It's also simpler for them not to have that many branches.

Reply:
@Hannibel 
<img width="371" alt="image" src="https://user-images.githubusercontent.com/503604/220070660-0bef0246-d74d-4eae-b2b3-af19e2f88bb4.png">
or here: 
<img width="892" alt="image" src="https://user-images.githubusercontent.com/503604/220070881-bcc1dcff-91c0-46ce-9ce2-6b05192da008.png">
or look at the timetable in our Slack channel.

@Ch4s3r Tagging commits is even more confusing/brittle/cumbersome for the candidate. With the branches, it is one interruption when the candidate is anyways already interrupted. E.g. by a colleague change, or because you'd need to look at new requirements.

Reply:
TBH I don't get why this PR is so controversial. This should be changed back to the **as-is state**: How we are doing this for years.

Reply:
@fleckb I did not mention to tag changes. Just add a `Iteration X:` before every commit and that's it.
Not tags, no branches.

Changing this back is ok, was just mentioning this as an alternative.

Reply:
We can remove the session, it is just more convenient to also have the session identifier. But maybe it's only me, since I use git CLI and I hate how most git UIs look. If there are 1000 different candidate branches I usually  just see something like this for example in which case I always have to go to github or slack to check the order of the sessions:
![image](https://user-images.githubusercontent.com/33929968/220168492-1cd16618-a38e-4ee8-9f85-a84f6fac5779.png)


> How we are doing this for years.

For some reason I thought we did with session identifier in the past, I'm not sure why 🤔 I mostly wanted to know what's your reasoning to remove it, since it actually can provide the benefit that based only on the branch name you could always order the progress.